pkgs/by-name/sc/scalpel/package.nix

```
@@ -0,0 +1,57 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, autoconf
+, automake
+, libtool
+, tre
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "scalpel";
+  version = "2.1";
+
+  src = fetchFromGitHub {
+    owner = "sleuthkit";
+    repo = "scalpel";
+    rev = "35e1367";
```

```suggestion
    rev = "35e1367ef2232c0f4883c92ec2839273c821dd39";
```
full hash required -- repo could go active and get millions of commits tmw, never know.